wire-library/wire-compiler/src/main/java/com/squareup/wire/ConsoleWireLogger.kt

```diff
@@ -31,15 +31,15 @@ internal class ConsoleWireLogger : WireLogger {
     }
   }
 
-  override fun artifactHandled(outputPath: Path, qualifiedName: String) {
+  override fun artifactHandled(outputPath: Path, qualifiedName: String, targetName: String) {
     if (quiet) {
       println(qualifiedName)
     } else {
-      println("Writing $qualifiedName to $outputPath")
+      println("target $targetName writing $qualifiedName to $outputPath")
     }
   }
 
-  override fun artifactSkipped(type: ProtoType) {
-    println("Skipping $type")
+  override fun artifactSkipped(type: ProtoType, targetName: String) {
+    println("target $targetName skipping $type")
```

Comment:
This will print as

```
target kotlin skipping squareup.common.CurrencyCode
```

I think I’d like something else?

```
Skipping squareup.common.CurrencyCode (target=kotlin)
```
?